examples/git_mirror.py

```diff
@@ -9,6 +9,7 @@ def main():
 
     target_ref = f"refs/heads/{branch_name}"
 
+    # if either src or dest require auth for read access, add the username and password args
```

Comment:
I'd add a commented out `src_{username,password}=<username,token>` and a commented out fetch_pack that uses them to make their comment easier to understand.